src/components/markdown/markdown-parser.ts

```diff
@@ -35,6 +36,17 @@ export async function markdownToHTML(
     }
 
     const file = await unified()
+        .use(() => {
+            return (tree: Node) => {
+                // Run the parse5.serialize function on all html elements,
+                // to minify it to get rid of extra whitespace that causes rendering issues
+                // when elements are opened/closed with empty lines between them.
+                visit(tree, 'html', (node: any) => {
+                    const documentFragment = parse5.parseFragment(node.value);
+                    node.value = parse5.serialize(documentFragment);
+                });
+            };
+        })
         .use(remarkParse)
         .use(remarkGfm)
```

Comment:
This should be placed after the first two functions in the pipeline:
        
```suggestion
    const file = await unified()
        .use(remarkParse)
        .use(remarkGfm)
        .use(() => {
            return (tree: Node) => {
                // Run the parse5.serialize function on all html elements,
                // to minify it to get rid of extra whitespace that causes rendering issues
                // when elements are opened/closed with empty lines between them.
                visit(tree, 'html', (node: any) => {
                    const documentFragment = parse5.parseFragment(node.value);
                    node.value = parse5.serialize(documentFragment);
                });
            };
        })
```

Reply:
Fixed, I did it together with a rebase with `main`.

_I was trying to move it around to see if it helped the issue 🤷‍♂️😅_